### DISCUSSION TITLE
MacOS build default password?

### DISCUSSION THREAD

**Discussion by Luppakorva** [awawa-dev/HyperHDR#193] [General]
Hi!

Sorry about the stupid question but I just cannot find it from the documents (been looking for an hour now), I'm surely just missing it. Installed via this: https://github.com/awawa-dev/HyperHDR/releases/download/v17.0.0.0/HyperHDR-17.0.0.0-macOS-x86_64.dmg 

Trying to open settings but I need password to log in. Found some mentioned "raspberry" for example elsewhere but no luck. 

localhost:8090 where I'm trying to login.

Could anyone let me know the right password for this? :) 

Thanks!

**Comment by awawa-dev** [comment]
Hi
Default installation for macOS or any other OS doesn't require any password to access web panel (you set it later). Unless you try to use some other imported configuration (copied to .hyperhdr folder).

**Comment by Luppakorva** [comment]
Thank you so much! Used my Hyperion installation credentials.  That sorted it out.

**Comment by awawa-dev** [comment]
You always can reset the password using command line running ```hyperhdr``` with proper switch on any OS (--help displays them all).
